tests/unit/lms/views/lti/basic_launch_test.py

```diff
@@ -297,19 +297,35 @@ def test__show_document_does_not_enable_grading_for_canvas(self, svc, context):
         context.js_config.enable_grading_bar.assert_not_called()
 
     @pytest.mark.usefixtures(
-        "with_gradable_assignment", "is_canvas", "with_student_grading_id"
+        "with_gradable_assignment",
+        "is_canvas",
+        "with_student_grading_id",
+        "user_is_learner",
     )
     def test__show_document_enables_speedgrader_settings(self, svc, context):
         svc._show_document(sentinel.document_url)  # pylint: disable=protected-access
 
-        context.js_config.add_canvas_speedgrader_settings(sentinel.document_url)
+        context.js_config.add_canvas_speedgrader_settings.assert_called_once_with(
```

Comment:
I've been yelling at the coverage report for a while until I notice this.

My personal favorite is asserting `assert_called_` themselves all the time.

Reply:
Oh FFS... yes good catch. I'm thinking it might be worth replacing this whole mess of fixtures by just setting the correct params to make each of these things happen. This way it would also document what we actually recieve, and should make it easy to parameterize.